lib/request-log.js

```diff
@@ -0,0 +1,59 @@
+var format = require('util').format;
```

Comment:
I'm trying out https://www.npmjs.com/package/hapi-bunyan to see if it suits our needs